Jenkinsfile

```diff
@@ -3,4 +3,7 @@ buildNPM {
   runLint = 'yes'
   runSonarqube = true
   runTest = 'yes'
+  runScripts = [
+   ['formatjs-compile': ''],
```

Comment:
looks like you need to add `"test": "yarn run test:jest && yarn run test:bigtest:ci"` and remove `runTest = 'yes'` to run scripts in parallel.

Reply:
@AndreiShumski2 thanks, it was added